src/devices/vdml_distance.c

```diff
@@ -0,0 +1,53 @@
+/**
+ * \file devices/vdml_imu.c
+ *
+ * Contains functions for interacting with the VEX Inertial sensor.
+ *
+ * Copyright (c) 2017-2019, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <errno.h>
+
+#include "pros/distance.h"
+#include "v5_api.h"
+#include "vdml/registry.h"
+#include "vdml/vdml.h"
+
+#define ERROR_DISTANCE_BAD_PORT(device, err_return)              \
+	if (vexDeviceDistanceStatusGet(device->device_info) == 0x82 || \
+	    vexDeviceDistanceStatusGet(device->device_info) == 0x82) { \
```

Comment:
These are testing against the same value. I don't think this is what you want.

Reply:
yikes, good catch. Need to change it to the other "good" value